web/src/settings_panel_menu.js

```diff
@@ -108,7 +108,7 @@ export class SettingsPanelMenu {
             // We need to re-register these handlers since they are
             // destroyed once the settings modal closes.
             this.org_user_settings_toggler.register_event_handlers();
-            this.set_key_handlers(this.org_user_settings_toggler, $("#admin-user-list"));
+            this.set_key_handlers(this.org_user_settings_toggler, $(".org-user-settings-switcher"));
```

Comment:
I was thinking of using an ID here, but toggle component doesn't allow custom ids, just a class and not sure if it's worth it to modify toggle for this change.

Reply:
These classes should be unique, so that's likely fine.